### PR TITLE
[antlr grammar comparison] integrated lexer evaluation in case of separate grammars

### PR DESCRIPTION
**Pull request by sailingKieler**
Signed-off-by: Christian Schneider <christian.schneider@itemis.de>

### CODE REVIEW

**Comment by sailingKieler** [comment]
Now ready for merge ... eventually

File: plugins/org.eclipse.xtext.generator/src/org/eclipse/xtext/generator/parser/antlr/XtextAntlrGeneratorComparisonFragment.xtend [hunk 1]
```diff
@@ -40,7 +43,7 @@ import org.eclipse.xtext.xtext.generator.parser.antlr.GrammarNaming
  * @author Christian Schneider - Initial contribution and API
  * @noreference
  */
-@Log
+ @Log
```
Comment (szarnekow) [inline plugins/org.eclipse.xtext.generator/src/org/eclipse/xtext/generator/parser/antlr/XtextAntlrGeneratorComparisonFragment.xtend]: You reverted my space-fix ;)

Comment (sailingKieler) [inline plugins/org.eclipse.xtext.generator/src/org/eclipse/xtext/generator/parser/antlr/XtextAntlrGeneratorComparisonFragment.xtend]: Ooops

Comment (oehme) [inline plugins/org.eclipse.xtext.generator/src/org/eclipse/xtext/generator/parser/antlr/XtextAntlrGeneratorComparisonFragment.xtend]: I thought MySpace went out of business?

Comment (szarnekow) [inline plugins/org.eclipse.xtext.generator/src/org/eclipse/xtext/generator/parser/antlr/XtextAntlrGeneratorComparisonFragment.xtend]: Apparently they are back :)

**Comment by szarnekow** [comment]
Jenkins test

**Pull request merged**